content/resources/how-to-guides/how-to-zapier-alert.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to Alert Task to Zapier Webhook
+description: >
+  Learn how to use webhooks with InfluxDB to send alerts to Zapier
+menu:
+  resources:
+    parent: How-to guides
+weight: 102
+---
+### How To: Alert Task to Zapier Webhook
+
+### By: Tyler Nguyen
+
+If you have been a long time user of InfluxDB, the role of tasks within the platform is, without a doubt, a
+
+spotlight feature of the product. Tasks allow users to automatically execute a bit of Flux code: 
+
+downsampling, transforming, or even creating data are all things that can be done with such a powerful tool 
+
+for automation. When dealing with potentially large amounts of raw data, automation is key to an efficient 
+
+workflow. Although we know tasks are extraordinarily useful and aid the process of automatically carrying 
+
+out repetitive jobs, there is still some degree of monitoring that needs to be done on the user end. 
+
+Fortunately, there is an incredibly simple way to take the automation one step further by using webhooks 
+
+combined with Zapier, a third party web app automation service, to propagate alerts to a host of different 
+
+applications and services with an effort to mitigate the level of micromanagement users must have when 
+
+performing certain tasks.
+
+
+## 1. Create a Zapier Webhook
+
+Zapier Webhooks are a premium feature, meaning you need to pay for the service. For a payment free 
+
+alternative, it is possible to use the InfluxDB application on the Zapier platform to send alerts for when 
+
+a bucket is listed or data is queried, if that fits your needs. 
+
+- Log into the Zapier Dashboard and find the “Create Zap” button
+
+-Name your Zap
+
+- In the “Trigger” panel, select the Webhook as an event trigger
+
+- There are three possible events: retrieving a poll, catching a raw hook, or just catching a hook. We will 
+
+choose the last option in this example.
+
+- Navigate to the “Set Up Trigger” where you can trigger on specific keys if you desire. In this example, we 
+
+will leave it blank and continue.
+
+- In the “Test Trigger” tab, you will be given your webhook, which you will need to save and give to InfluxDB 
```

Comment:
If something is a step-by-step process, use an ordered list instead of an unordered list.

```suggestion
1. Log into the Zapier Dashboard and click the **Create Zap** button.

2. Name your Zap.

3. In the **Trigger** panel, select **Webhook** as an event trigger.
    There are three possible events:

        - retrieving a poll
        - catching a raw hook
        - catching a hook
        
        For this example, select **catching a hook**.

4. _(Optional)_ Navigate to **Set Up Trigger**.
    Zapier lets you trigger on specific keys in your web request.
    For this example, leave it blank and continue.

5. In the **Test Trigger** tab, Zapier provides your webhook URL.
    Copy and store this URL. 
```